### ISSUE TITLE
truncate error

### ISSUE THREAD

**Issue by gotoin** [cakephp/phinx#1141]
```php
public function run()
    {
        $data = array(
            array(
                'user_id'   => 2,
                'created'   => date('Y-m-d H:i:s', time()),
                'update'    => time(),
            ),
            array(
                'user_id'   => 3,
                'created'   => date('Y-m-d H:i:s', time()),
                'update'    => time(),
            ),
            array(
                'user_id'   => 4,
                'created'   => date('Y-m-d H:i:s', time()),
                'update'    => time(),
            ),
            array(
                'user_id'   => 5,
                'created'   => date('Y-m-d H:i:s', time()),
                'update'    => time(),
            ),
        );

        $table = $this->table('test');
        $table->insert($data);
        $table->save();

        $table->truncate();
    }
```

the error message:

```
PHP Fatal error:  Uncaught Error: Cannot call abstract method Phinx\Db\Adapter\AdapterInterface::truncateTable() in /Users/www/vendor/robmorgan/phinx/src/Phinx/Db/Adapter/TablePrefixAdapter.php:105
Stack trace:
#0 /Users/www/vendor/robmorgan/phinx/src/Phinx/Db/Table.php(662): Phinx\Db\Adapter\TablePrefixAdapter->truncateTable('test')
#1 /Users/www/db/seeds/TestSeeder.php(44): Phinx\Db\Table->truncate()
#2 /Users/www/vendor/robmorgan/phinx/src/Phinx/Migration/Manager/Environment.php(154): TestSeeder->run()
#3 /Users/www/vendor/robmorgan/phinx/src/Phinx/Migration/Manager.php(395): Phinx\Migration\Manager\Environment->executeSeed(Object(TestSeeder))
#4 /Users/www/vendor/robmorgan/phinx/src/Phinx/Migration/Manager.php(514): Phinx\Migration\Manager->executeSeed('dev', Object(TestSeeder))
#5 /Users/www/vendor/robmorgan/phinx/src/Phinx/Console/Command/SeedRun.php(110): Phinx\Migration\Manager->seed('dev')
#6 /Users/ in /Users/www/vendor/robmorgan/phinx/src/Phinx/Db/Adapter/TablePrefixAdapter.php on line 105

Fatal error: Uncaught Error: Cannot call abstract method Phinx\Db\Adapter\AdapterInterface::truncateTable() in /Users/www/vendor/robmorgan/phinx/src/Phinx/Db/Adapter/TablePrefixAdapter.php:105
Stack trace:
#0 /Users/www/vendor/robmorgan/phinx/src/Phinx/Db/Table.php(662): Phinx\Db\Adapter\TablePrefixAdapter->truncateTable('test')
#1 /Users/www/db/seeds/TestSeeder.php(44): Phinx\Db\Table->truncate()
#2 /Users/www/vendor/robmorgan/phinx/src/Phinx/Migration/Manager/Environment.php(154): TestSeeder->run()
#3 /Users/www/vendor/robmorgan/phinx/src/Phinx/Migration/Manager.php(395): Phinx\Migration\Manager\Environment->executeSeed(Object(TestSeeder))
#4 /Users/www/vendor/robmorgan/phinx/src/Phinx/Migration/Manager.php(514): Phinx\Migration\Manager->executeSeed('dev', Object(TestSeeder))
#5 /vendor/robmorgan/phinx/src/Phinx/Console/Command/SeedRun.php(110): Phinx\Migration\Manager->seed('dev')
#6 /Users/ in /Users/www/vendor/robmorgan/phinx/src/Phinx/Db/Adapter/TablePrefixAdapter.php on line 105
```

thanks

**Comment by lorenzo** [comment]
What version are you using?

**Comment by gotoin** [comment]
"robmorgan/phinx": "^0.8.1"

**Comment by MasterOdin** [comment]
@dereuromark this issue can be closed. The issue was fixed in #1111 which was part of the 0.9.0 release

**Comment by othercorey** [comment]
Fixed in https://github.com/cakephp/phinx/pull/1111
